apps/frontend/src/libs/maps/icon-name-to-svg.map.ts

```diff
@@ -0,0 +1,10 @@
+import wheelOfBalanceActive from "~/assets/img/wheel-of-balance-active.svg";
+import wheelOfBalanceIncative from "~/assets/img/wheel-of-balance-inactive.svg";
+import { IconName } from "~/libs/types/types.js";
+
+const iconNameToSvg = new Map<IconName, string>([
+	["my-wheel-active", wheelOfBalanceActive],
+	["my-wheel-inactive", wheelOfBalanceIncative],
+]);
```

Comment:
If we want to store the values as a react element which is an icon, we'll have to install vite-plugin-svgr, and create a separate folder in assets/img for icons, after that configuring the vite-plugin-svgr to target this folder specifically. Otherwise, all svg imports in project will be treated as React.FC<React.SVGProps<SVGSVGElement>>>, which might not be a good behaviour

Reply:
<img width="1440" alt="Screenshot 2024-08-24 at 15 14 17" src="https://github.com/user-attachments/assets/3d444e10-a7af-41a2-8c46-9a668da666e7">

Reply:
Keep `img`, I will create a separate ticket for adding `vite-plugin-svgr`